flox-bash/tests/multi-env.bats

```diff
@@ -0,0 +1,66 @@
+#! /usr/bin/env bats
+# -*- mode: bats; -*-
+# ============================================================================ #
+#
+# Test the `FLOX_ENV' variable set by `flox activate' is set appropriately when
+# multiple environments are indicated on the CLI.
+#
+# ---------------------------------------------------------------------------- #
+
+bats_load_library bats-support;
+bats_load_library bats-assert;
+bats_require_minimum_version 1.5.0;
+
+load test_support.bash;
+
+
+# ---------------------------------------------------------------------------- #
+
+kill_envs() {
+  "$FLOX_CLI" destroy -e "${TEST_ENVIRONMENT}1" --origin -f||:;
+  "$FLOX_CLI" destroy -e "${TEST_ENVIRONMENT}2" --origin -f||:;
+}
+
+setup_file() {
+  common_setup;
+  kill_envs;
+}
+
+teardown_file() {
+  kill_envs;
+}
+
+
+# ---------------------------------------------------------------------------- #
+
+@test "init flox ${TEST_ENVIRONMENT}{1,2}" {
```

Comment:
If the goal is parallelizing, it seems like you should set TEST_ENVIRONMENT to something unique to this file, or it might be more clear to use a separate variable

Reply:
sure those could be longer or something that's a good idea.